prow/scripts/provision-vm-compass.sh

```diff
@@ -16,31 +16,69 @@ source "${SCRIPT_DIR}/lib/utils.sh"
 # shellcheck source=prow/scripts/lib/gcp.sh
 source "$SCRIPT_DIR/lib/gcp.sh"
 
+# Download yq
+YQ_VERSION="v4.25.1"
+log::info "Downloading yq version: $YQ_VERSION"
+wget "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64"
+mv yq_linux_amd64 yq
+chmod +x yq
+
+# Flag to recycle or recreate VM on each commit execution
+RECYCLE_VM="no"
+PROVISIONING_PASS="no"
```

Comment:
Can you put a more meaningful name than `PROVISIONING_PASS` as it doesn't tell me anything just looking at it AND/OR add a comment explaining it like with the `RECYCLE_VM`.